### PR TITLE
feat: Allow naming of result PDF

### PR DESCRIPTION
**Pull request by dgboss**
Add optional result_name parameter to run method.

### CODE REVIEW

**Review by ianschmitz — CHANGES_REQUESTED** [review]
Looks good. We'll need to update the version number before merging: https://github.com/geocortex/vertigis-reporting-client-py/blob/a9a2d7e3ea550f7de03370ac76ce9b7e84c358cf/setup.py#L19

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ url = await run("itemid", [... other arguments])
 | portal_url     | str  | The URL of the ArcGIS Portal instance to use. Defaults to ArcGIS Online: `"https://www.arcgis.com"`                                                                                                                            |
 | token          | str  | The Portal access token to be used to access secured resources. If not provided requests to secured resources will fail.                                                                                                       |
 | culture        | str  | The culture to use for localization. For example `"en-US"`.                                                                                                                                                                    |
+| result_file_name    | str  | The desired name of the output file.  |
```
Comment (ianschmitz) [inline README.md:39]: Do you mind updating the docs here and in code to what you came up with on the JS side?

Comment (dgboss) [reply README.md]: I've made it similar, but omitted the part about the browser tab as that is not likely relevant.

**Comment by dgboss** [comment]
@ianschmitz Version 1.1.0 or 1.0.1?

**Comment by ianschmitz** [comment]
> @ianschmitz Version 1.1.0 or 1.0.1?

`1.1.0`

**Review by ianschmitz — APPROVED** [review]

**Pull request merged**